src/main/java/frc/trigon/robot/components/XboxController.java

```diff
@@ -71,11 +68,10 @@ public void setDeadband(double deadband) {
     }
 
     public void rumble(double durationSeconds, double power) {
-        if (stopRumblingCommand != null)
-            stopRumblingCommand.cancel();
-        stopRumblingCommand = Commands.getDelayedCommand(durationSeconds, () -> getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0));
-        stopRumblingCommand.schedule();
-        getHID().setRumble(GenericHID.RumbleType.kBothRumble, power);
+        new StartEndCommand(
+                () ->  getHID().setRumble(GenericHID.RumbleType.kBothRumble, power),
+                () -> getHID().setRumble(GenericHID.RumbleType.kBothRumble, 0)
+        ).withTimeout(durationSeconds).schedule();
```

Comment:
Hmmmm, what about a case where this was called while another command is running?

Reply:
Honestly, that's probably why I did it as I did previously.

Reply:
Now that I'm thinking about it, would that really be a problem? The old command might stop it for a split second but the rumble in the new command is a runnable so it'll just get right back to rumbling

Reply:
We should choose the second option. And that's not how a _start_ end command works. It runs something when it starts, and something when it ends. 
Say I rumbled for 4 seconds. 
2 seconds later, I rumbled for 5 seconds. 
After 2 more seconds the rumbling will stop.

Reply:
Okay, I've read the docs and it runs "once" so nvm

I'll implement a fix as soon as I get home